tests/integration/component.go

```diff
@@ -159,6 +159,25 @@ func componentTests(args ...string) {
 			helper.MatchAllInOutput(cmpList, []string{"cmp-git", "Not Pushed"})
 			helper.CmdShouldPass("odo", append(args, "delete", "-f", "--all", "--context", commonVar.Context)...)
 		})
+		Context("when components are not created/managed by odo", func() {
+			var appName, compName string
+			JustBeforeEach(func() {
+				//CreateNonOdoComponent
```

Comment:
Can someone help me with this one? I plan on creating a deployment and deployment-config manually via API, but I need to prepare a lot of data before doing that. Do we already have helper functions that might help here?

Reply:
you can just create `deployment.yaml` with the most simple deployment that you can come up with. Add the same labels to it as WebConsole does and just call `kubectl apply -f deplyomet.yaml`